frontend/src/App.tsx

```diff
@@ -17,7 +17,8 @@ function App() {
     <div className="app-container">
       <Routes>
         <Route path="/" element={<Layout />}>
-          <Route index element={<Candidates />} />
+          <Route index element={<ApplyPage />} />
```

Comment:
Its okay to have this element at the index when testing an creating the page. But it should be on a separate route like on line 22 when we put out the PR

The routes would be the routes that the prescreening navigates to. 

Looking at the code of the `<ApplyPage />` it seems like that page is redundant work and what is returned there should be the element given the the path. It that make sense so `<Route path="path1" element={<PathhAppLayout path={1}/>} />` for example